tests/test_dht_schema.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from hivemind.dht import get_dht_time
+from hivemind.dht.node import DHTNode, LOCALHOST
+from hivemind.dht.schema import SchemaValidator
+
+
+@pytest.mark.forked
+@pytest.mark.asyncio
+async def test_schema_validator():
+    validator = SchemaValidator({
+        b'experiment_name': {'type': 'binary'},
+        b'n_batches': {'type': 'dict',
+                       'keysrules': {'type': 'binary'},
+                       'valuesrules': {'type': 'integer', 'min': 0}},
+        b'signed_data': {'type': 'dict',
+                         'keysrules': {'type': 'binary', 'regex': rb'^.*\[owner:.+\].*$'}},
```

Comment:
It might be possible to avoid these custom formats (which might not be familiar to users without Cerberus experience) with [pydantic](https://github.com/samuelcolvin/pydantic/) and native annotations